src/collar/oc_settings.lsl

```diff
@@ -1,553 +1,595 @@
-// This file is part of OpenCollar.

```

Comment:
What is the change in this file? It is showing that the entire file was changed.